.github/workflows/unit-tests.yml

```diff
@@ -0,0 +1,76 @@
+name: unit-tests
+
+on:
+  pull_request:
+  push:
+    branches:
+      - "master"
+      - "open-release/*.master"
+  
+jobs:
+  run-tests:
+    if: (github.repository != 'openedx/edx-platform' && github.repository != 'edx/edx-platform-private') || (github.repository == 'openedx/edx-platform' && (startsWith(github.base_ref, 'open-release') == true))
+    runs-on: [ ubuntu-20.04 ]
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [3.8]
+        django-version:
+          - "pinned"
```

Comment:
We won't be using other Django versions than the pinned one, so we can remove this.

Reply:
Sure. This is removed and django version is handled based on edx-platform-version.